docs: #775 Changed danger admonition in Console doc (#778)
redpanda-data/documentation: docs/platform/console/reference/config.mdx

### Environment variables

Configuration options can be configured using environment variables. The key for the environment
variable is auto-generated by converting the YAML equivalent to uppercase and adding an underscore for each indentation level. For example:

- kafka.rackId => KAFKA_RACKID
- kafka.tls.caFilepath => KAFKA_TLS_CAFILEPATH

For configuration properties that expect a list of values, use commas between each value. For example:

- KAFKA_BROKERS=redpanda-0:9092,redpanda-1:9092,redpanda-2:9092

:::important
You cannot use environment variables to configure object arrays, such as the configuration
for Kafka connect clusters. In this case, use a YAML file, and provide secrets using environment variables or command line arguments.
:::

## Sample configuration

The following `config.yaml` configuration file contains a complete list of all Console configuration properties and their descriptions. All values are default values.

Provide the filepath to your configuration file by setting either the flag
`-config.filepath` or the environment variable `CONFIG_FILEPATH`.

:::note

> This configuration file contains both enterprise and community configurations. If you don't provide an enterprise license, Console ignores configurations for enterprise features.

Reviewer: ```suggestion
This configuration file contains both Redpanda Enterprise and Redpanda Community configurations. If you don't provide a Redpanda Enterprise license, Console ignores configurations for Redpanda Enterprise features.
```

  # Client ID that the Kafka client can use to identify itself 
  # against the target Kafka cluster.
  clientId: console
  # In multi-zone Kafka clusters, you can reduce traffic costs 
  # by consuming messages from replica brokers in the same zone
  rackId:
  # SASL configuration for Console to connect to the Kafka API.
  sasl:
    enabled: false
    username:

>     # Password can also be set via the --kafka.sasl.password flag.

Reviewer: per RP Style Guide, no "via":
# Password can also be set using the --kafka.sasl.password flag.

      kerberosConfigPath:
      serviceName:
      username:

>       # Password can also be set via the --kafka.sasl.gssapi.password flag

Reviewer: ```suggestion
      # Password can also be set using the --kafka.sasl.gssapi.password flag
```

      password:
      realm:
      enableFast: true
    oauth:

>       # can also be set via the --kafka.sasl.oauth.token flag

Reviewer: ```suggestion
      # can also be set via the --kafka.sasl.oauth.token flag
```

Won't mark this moving forward--just do a global search and replace of "via". Thx

# can only be used with a valid Enterprise license.
enterprise:
  rbac:

>     # Whether or not RBAC will be used. This must be enabled

Reviewer: ```suggestion
    # Whether or not RBAC is used. This must be enabled
```
